gateway/src/apicast/proxy.lua

```diff
@@ -300,10 +302,19 @@ function _M:rewrite(service, context)
   context.credentials = ctx.credentials
 end
 
-function _M:access(service, usage, credentials, ttl)
+function _M:access(service)
```

Comment:
Why did you remove the params? The call from `apicast.lua` still sends them: https://github.com/3scale/APIcast/blob/f02d7d89c521db910c7bf885b13097cbf227a3ae/gateway/src/apicast/policy/apicast/apicast.lua#L100

I'm not sure how this affects the return below where these params are ignored.